src/Fixer/Phpdoc/PhpdocToCommentFixer.php

```diff
@@ -91,6 +179,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            if (0 < Preg::matchAll('~\@([a-zA-Z0-9_\\\\-]+)(?=\s|\(|$)~', $token->getContent(), $matches)) {
```

Comment:
Can't we replace the lookahead with `\b`?

Reply:
Sorry, I don't know a lot about regex. I created this one by copying some part from the `CommentToPhpdocFixer`.

What do you mean ?

Reply:
The change below in suggestion (I hoped GitHub will treat is as replay here as it's on the same line).